lib/msal-browser/docs/v1-migration.md

```diff
@@ -111,10 +111,10 @@ async function signIn(method) {
 
 async function getTokenPopup(request) {
     return await myMSALObj.acquireTokenSilent(request).catch(async (error) => {
-        console.log("silent token acquisition fails. acquiring token using popup");
+        this.logger.info("silent token acquisition fails. acquiring token using popup");
         // fallback to interaction when silent call fails
         return await myMSALObj.acquireTokenPopup(request).catch(error => {
-            console.log(error);
+            handleError(error);
```

Comment:
Are we adding a note what `handleError(error)` should be?

Reply:
@pkanher617 Do you have any input? After our discussion on Friday I just changed it to just handleError, but am happy to put something else that is more descriptive